OpenRA.Mods.Common/Traits/SelfHealing.cs

```diff
@@ -18,7 +18,8 @@ class SelfHealingInfo : UpgradableTraitInfo, Requires<HealthInfo>
 	{
 		public readonly int Step = 5;
 		public readonly int Delay = 5;
-		public readonly float HealIfBelow = .5f;
+		[Desc("Heal if current health is below this percentage of full health.")]
+		public readonly int HealIfBelow = 50;
```

Comment:
I'd say it's almost a tie between `Percent` and `Percentage` currently, plus a couple of cases like this with neither.